### PR TITLE
vcsh: add page

### PR DESCRIPTION
**Pull request by derphilipp**
Adds tldr page for cli tool vcsh, see https://github.com/RichiH/vcsh

----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/vcsh.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# vcsh
+
+> Version Control System for $HOME - multiple Git repositories in $HOME.
```
Comment (mfrw) [inline pages/common/vcsh.md]: enclose $HOME in backtics => `$HOME`

File: pages/common/vcsh.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Initialize (empty) repository:
+`vcsh init {{repository_name}}`
+
+- Clone repository into custom name:
+`vcsh clone {{git_url}} {{repository_name}}`
+
+- List all managed repositories:
+`vcsh list`
+
+- Pull/pull all managed repositories from/to remotes:
```
Comment (mfrw) [inline pages/common/vcsh.md]: Perhaps `Push or pull` reads better.

File: pages/common/vcsh.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Clone repository into custom name:
+
+`vcsh clone {{git_url}} {{repository_name}}`
+
+- List all managed repositories:
+
+`vcsh list`
+
+- Pull/pull all managed repositories from/to remotes:
```
Comment (pxgamer) [inline pages/common/vcsh.md]: I'm assuming this should be `Push/pull all...`?

File: pages/common/vcsh.md (hunk 1 above)
Comment (derphilipp) [reply pages/common/vcsh.md]: Agreed!

File: pages/common/vcsh.md (hunk 3 above)
Comment (derphilipp) [reply pages/common/vcsh.md]: Thank you for noticing! I fixed it;

File: pages/common/vcsh.md (hunk 2 above)
Comment (derphilipp) [reply pages/common/vcsh.md]: I think due to the sentence structure

Push or pull all managed repositories from or to remotes
Push/pull all managed repositories from/to remotes

the second one is more readable as the dashes indicate a selection better with the "from/to"; and I would prefer it over "from or to" - but I do not insist on it

File: pages/common/vcsh.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# vcsh
+
+> Version Control System for `$HOME` - multiple Git repositories in `$HOME`.
```
Comment (agnivade) [inline pages/common/vcsh.md]: I am not exactly clear what this does. The `-` in the middle is a bit odd. Can you explain a bit ?

File: pages/common/vcsh.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Execute git command on managed repository:
+
+`vcsh {{repository_name}} {{git_command}}`
+
+- Initialize (empty) repository:
```
Comment (agnivade) [inline pages/common/vcsh.md]: We usually list commands in the order they are used. I think a user will `init` the repo first ? So we should mention init first, then others.

Comment (derphilipp) [reply pages/common/vcsh.md]: Okay, i changed it from 'most used' to 'order of first use'

File: pages/common/vcsh.md (hunk 4 above)
Comment (derphilipp) [reply pages/common/vcsh.md]: I changed the description from the official title to a more descriptive one.

**Comment by agnivade** [comment]
Thanks. Can you take a look at this [comment](https://github.com/tldr-pages/tldr/pull/2338#pullrequestreview-156244513) of mime ?

**Comment by pxgamer** [comment]
`$home` in Windows is `%USERPROFILE%`

File: pages/common/vcsh.md (hunk 2 above)
Comment (mfrw) [reply pages/common/vcsh.md]: yes, i think the second one seems fine.

**Pull request merged**